env.d/development/dev.dist

```diff
@@ -5,7 +5,9 @@ AUTHENTICATION_BACKEND=dummy
 # LMS Backend
 EDX_BASE_URL=http://localhost:8073
 EDX_BACKEND=richie.apps.courses.lms.base.BaseLMSBackend
+EDX_COURSE_REGEX='^.*/courses/(?P<course_id>.*)/info/?$'
+EDX_JS_COURSE_REGEX='^.*/courses/(.*)/info/?$'
```

Comment:
It look's like a other topic but i'm not sure that it deserve a commit.

Reply:
Yup, I could move it into a dedicated commit you're right